setup/README.md

```diff
@@ -124,134 +172,83 @@ If you were just trusting everything someone told you, you wouldn't be here look
 
 `kubectl get all -n connaisseur` will show you most* of what you deployed (ConfigMaps, Secrets and MutatingWebhookConfiguration are not shown).
 
-### 1. Deploy (un)signed images
+### Deploy (un)signed images
+
+Let's test if our configuration works. We need to prepare a signed and an unsigned image and push these to a remote repository. For simplicity, we define an image path variable (host, repository and image). Below, substitute `IMAGE_PATH` with your own the values as appropriate:
+
+```bash
+# Typically, IMAGE_PATH=<YOUR-REGISTRY>/<YOUR-REPOSITORY-NAME-/-DOCKER-HUB-ID>/<IMAGE-NAME>
+IMAGE_PATH=docker.io/connytest/testimage
```

Comment:
Who is the owner of this repo?

Reply:
Me. It is also the one I use in the demo. It all ties together and you are safe if you use that value and the sample key below ;-)
Also created #41 about this. What is bad if we move it into SSE, is that it makes every command like this much longer...just from a UX perspective.
What do you think?

Reply:
Oh, didn't see that :+1: I think it would be cleaner if it was moved into SSE. Also, I don't think the long path matters much. However, if we decide to move it we can do it independently of this PR I guess :)

Reply:
yes, that was my rationale as well...